response_operations_social_ui/views/social/generate_iac.py

```diff
@@ -13,11 +13,15 @@
 @login_required
 def generate_iac():
     case_id = request.form['case_id']
-
     new_iac = case_controller.generate_iac(case_id)
+
     context = build_view_social_case_context(case_id)
     logger.debug("generate_iac", case_id=case_id)
 
-    context['new_iac'] = new_iac
+    def split_string(x, n):
+        return [x[i:i + n] for i in range(0, len(x), n)]
+    split_hac = split_string(new_iac, 4)
+    formatted_hac = (' '.join(map(str, split_hac)))
+    context['new_iac'] = formatted_hac
```

Comment:
Do we need a test for this? What happens when generate_iac returns a non-string type?